editor/bundle/src/main/webapp/clientlibs-root/io.wcm.caconfig.editor/js/widgets/widgets.constants.js

```diff
@@ -20,7 +20,7 @@
 (function (angular) {
   "use strict";
 
-  var ANYTHING = /^.*$/;
+  var ANYTHING = /.*/;
```

Comment:
Not sure about this.  the current one would not work since it would invalidate any text with more than one line. The textfield does not support any line break at all, so maybe it is fine to change it?